feinstaub/nginx.conf.sigil

```diff
@@ -107,8 +109,7 @@ upstream {{ $.APP }}-{{ $upstream_port }} {
 {{ range $listeners := $.DOKKU_APP_LISTENERS | split " " }}
 {{ $listener_list := $listeners | split ":" }}
 {{ $listener_ip := index $listener_list 0 }}
-  server {{ $listener_ip }}:{{ $upstream_port }};{{ end }}
+{{ $listener_port := index $listener_list 1 }}
+  server {{ $listener_ip }}:{{ $listener_port }};{{end}}
```

Comment:
Why have we changed this?

Reply:
Ping @karimkawambwa .

Reply:
I have not come across a reason for this change, so i have reverted it.